src/cartesian/Area.js

```diff
@@ -263,13 +264,13 @@ class Area extends Component {
       labelItem = option(props);
     } else {
       labelItem = (
-        <text
+        <Text
           key={props.key}
           {...getPresentationAttributes(props)}
```

Comment:
`getPresentationAttributes(props)` can be replaced by `props`, because `Text` is no longer a simple svg element.  `getPresentationAttributes` should be done in `Text`.